cmd/cri-containerd/cri_containerd.go

```diff
@@ -118,3 +122,28 @@ func validateConfig(o *options.CRIContainerdOptions) {
 		selinux.SetDisabled()
 	}
 }
+
+func setupDumpStacksTrap() {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, syscall.SIGUSR1)
+	go func() {
+		for range c {
+			dumpStacks()
+		}
+	}()
+}
+
+func dumpStacks() {
```

Comment:
from: https://golang.org/src/runtime/debug/stack.go, just a little cleaner with a modification to dump all (true)
```
	buf := make([]byte, 1024)
  	for {
  		n := runtime.Stack(buf, true)
  		if n < len(buf) {
  			return buf[:n]
  		}
  		buf = make([]byte, 2*len(buf))
  	}
```

Reply:
Done.